Blockchain/readme.md

```diff
@@ -115,4 +120,4 @@
   - [Austin Griffith](https://www.youtube.com/channel/UC_HI2i2peo1A-STdG22GFsA)
   - [Nader Dabit](https://www.youtube.com/user/boyindasouth)
   - [Harkirat Singh (Youtube)](https://www.youtube.com/playlist?list=PLVKLWop9wWA82pZoyylZD2VF2c7MR8_5I)
-  - [Daulat Hussain (Youtube)](https://www.youtube.com/@daulathussain/featured)
```

Comment:
Would you like to explain why did you remove this YT link?

Reply:
That line was coming in conflicts! sorry if I made any mistake. I just want to add the resources I have mentioned. please tell me what are conflicts coming?

Reply:
So should I close this PR and open a new one?

Reply:
Your PR, so whatever way you are comfortable with.